test/e2e/reconciler/cache/replication_test.go

```diff
@@ -678,14 +683,23 @@ func (b *replicateResourceScenario) verifyResourceReplicationHelper(ctx context.
 		}
 		unstructured.RemoveNestedField(originalResource.Object, "metadata", "resourceVersion")
 		unstructured.RemoveNestedField(cachedResource.Object, "metadata", "resourceVersion")
+
+		// TODO(davidfestal): find out why the generation is not equal, specially for rbacv1.
```

Comment:
OK. I think I found it: https://github.com/kcp-dev/kubernetes/pull/132

Once this PR merges we need to set the field to `true` in https://github.com/kcp-dev/kcp/blob/0d6a2ae4b61405bbfc8cfa6472e316c6b8298b95/pkg/cache/server/config.go#L203

Reply:
I think we could wait until the PR merges and test it in your PR, wdyt?